packages/@o3r/test-helpers/src/prepare-test-env.ts

```diff
@@ -149,8 +149,10 @@ export async function prepareTestEnv(folderName: string, options?: PrepareTestEn
 
   prepareFinalApp(appDirectory);
 
-  // Setup git and initial commit to easily make checks on the diff inside the tests
-  setupGit(workspacePath);
+  if (type !== 'blank') {
```

Comment:
Why this change?
Actually this git init is for comparing the code generated post env setup and it would make sense in case of blank env as well :S.

Reply:
I see, so maybe it would make more sens to add an option to the env setup function to explicitly not initialize a git env when it is not required.
The purpose is to still allow the git diff functions in this env for future tests that may not explicitly initialize a new git repository